webpack.config.js

```diff
@@ -414,6 +414,7 @@ var aliases = {
     "d3": "d3/d3.js",
     "moment": "moment/moment.js",
     "react": "react-lite/dist/react-lite.js",
+    "react-dom": "react-lite/dist/react-lite.js",
```

Comment:
Are you using this anywhere?

Reply:
Indirectly - the patternfly-react needs it.
The patternfly-react dist contains just the patternfly-react sources. Its dependencies are just listed in package.json and is up to consuming webpack to link it.